bin/sdk/gutenberg.js

```diff
@@ -10,6 +10,7 @@ exports.config = ( {
 } ) => {
 	const baseConfig = getBaseConfig( {
 		cssFilename: '[name].css',
+		cssModules: true,
```

Comment:
What's the goal with this? Will it be possible for css-modules style generated CSS and our current monolithic CSS to co-exist?